provider/cmd/pulumi-resource-docker/schema.json

```diff
@@ -3039,6 +3039,11 @@
                     },
                     "description": "An optional map of named build-time argument variables to set during the Docker build. This flag allows you to pass built-time variablesthat can be accessed like environment variables inside the RUN instruction."
                 },
+                "builderVersion": {
+                    "type": "string",
```

Comment:
Should we make this an enum instead? https://www.pulumi.com/docs/guides/pulumi-packages/schema/#complextype